sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubOptions.cs

```diff
@@ -132,6 +134,48 @@ public int MaxEventBatchSize
             }
         }
 
+        private int _minEventBatchSize;
+
+        /// <summary>
+        /// Gets or sets the minimum number of events delivered in a batch. This setting applies only to functions that
+        /// receive multiple events. This value must be less than <see cref="MaxEventBatchSize"/>. This value is used in
+        /// conjunction with <see cref="MaxWaitTime"/>. Default 1.
+        /// </summary>
+        public int MinEventBatchSize
+        {
+            get => _minEventBatchSize;
+
+            set
+            {
+                if (value < 0)
+                {
+                    throw new ArgumentException("Batch size must be larger than or equal to 0.");
```

Comment:
Do we want to validate min < max? Seems impossible unless we write a constructor

Reply:
The validation can be added [here](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubWebJobsBuilderExtensions.cs#L89) as a PostConfigure step.

Reply:
Should be addressed now :)